### ISSUE TITLE
Memory game disappear animation leaves ghosts

### ISSUE THREAD

**Issue by cdlm** [pharo-graphics/Tutorials#12]
When you find a pair, cards are animated scaling smaller and getting transparent.
They're supposed to become fully invisible but they don't, and the ghost cards left behind are still interactive.

<img width="532" alt="Solved memory game with zombie cards" src="https://user-images.githubusercontent.com/136339/233717990-6b099233-6555-43fc-9492-90451caccbb7.png">

In the screenshot, the game is fully solved. The small square in the rightmost column is an example of a card that didn't fully disappear. There are other smaller ones in the first row. The code is supposed to scale the cards down to 1% of their full size which is much smaller than that.

The two big ghost cards in the middle are what you get by clicking those tiny squares.

To me this suggests either that the animation stops before it's really finished, or that when it stops it doesn't get a chance to update the card elements to their final state.

**Comment by tinchodias** [comment]
This is probably the same cause as this bug: https://github.com/pharo-graphics/Bloc/issues/194

**Comment by cdlm** [comment]
Indeed, that's quite possible. It's like the penultimate frame is drawn, the animation finishes before the next one, and there should be (but actually isn't) an additional frame drawn to show the final state of the animation even if it's drawn a bit late.
